src/components/Features/Account/Account.module.scss

```diff
@@ -0,0 +1,4 @@
+.accountMenu {
+  padding: 8px;
```

Comment:
Can we add this styling to `Menu` instead of defining this here, in the `SelectToken` and in `Source`?

Reply:
selectToken and source have only the 16px from the bridge. Account menu needs 8px in addition (24px in Figma)